src/Evolution/DiscontinuousGalerkin/Actions/InternalMortarDataImpl.hpp

```diff
@@ -173,8 +173,7 @@ void internal_mortar_data(
       // optimization. This will require making the
```

Comment:
typo in commit message computed -> compute

Reply:
In the future, could you copy-paste part of the beginning of the commit message? This comment shows up in a few different commits